web/test/pages/profile.test.tsx

```diff
@@ -1727,6 +1727,29 @@ describe("profile", () => {
     });
   });
 
+  describe("Information Section", () => {
+    describe("business structure field", () => {
+      it("displays Limited Liability Company (LLC) when legal structure id is limited-liability-company", () => {
+        const legalStructure = "limited-liability-company";
+
+        renderPage({
+          userData: generateUserData({
+            formationData: generateFormationData({
```

Comment:
Is there a reason to why you had to explicitly set the getFilingResponse to true because the test still passes if when I deleted it.

Reply:
Yes, b/c it will always pass otherwise. I implemented the test first and with red-green factoring, the test was passing on implementation. When I added the formationData, the test failed. Once I implemented my code, then the test passed.